tests/cypress/template/navbar-languages.cy.js

```diff
@@ -0,0 +1,12 @@
+describe('Navbar languages', () => {
+
+  it('Sets and reads SKOSMOS_LANGUAGE cookie correctly', () => {
+    cy.visit('/')
+
+    cy.get('.nav-item.language a').first().click()
+    cy.getCookie('SKOSMOS_LANGUAGE').should('exist')
+    cy.getCookie('SKOSMOS_LANGUAGE').should('have.property', 'value', 'fi')
+
+    cy.url().should('include', '/fi/')
+  })
```

Comment:
How about also visiting the site root `/` again to verify that the cookie had an effect?